locales/en/messages.json

```diff
@@ -3120,6 +3120,39 @@
     "pidTuningNotchFilterHelp": {
         "message": "The Notch Filter has a Center and a Cutoff. The filter is symmetrical. The Center Frequency is the center of the filter and the Cutoff Frequency is where Notch filter starts. For example with Notch Cutoff of 160 and Notch Center of 260 it means the range is 160-360Hz with most attenuation around center"
     },
+    "pidTuningDynamicNotchFilterGroup": {
+        "message": "Dynamic Notch Filter"
+    },
+    "pidTuningDynamicNotchFilterHelp": {
+        "message": "Dynamic Notch Filter tracks peak motors noise frequency and places one or two notch filters with their center at that frequency."
+    },
+    "pidTuningDynamicNotchFilterDisabledWarning": {
+        "message": "<span class=\"message-negative\">Warning:</span> Dynamic Notch Filter is disabled, please enable it as a feature in the Configuration tab to show related settings."
```

Comment:
This gives the impression that the user **has to** enable the dynamic notch filter, which is not what I think this should be. I'd rather go for a notice like:

Notice: The dynamic notch filter is disabled. In order to configure and use it, please enable the 'DYNAMIC_FILTER' feature in the 'Other Features' section of the 'Configuration' tab.

Reply:
Yeah, changed it...